fil-proofs-tooling/src/bin/benchy/main.rs

```diff
@@ -1,3 +1,6 @@
+//requires nightly, or later stable version
+//#![warn(clippy::unwrap_used)]
```

Comment:
This might be possible now. I haven't checked though.

Reply:
That surely isn't a blocker to get this merged.

Reply:
I think it works fine on later stable, definitely not on the toolchain we're using though.  If/when we upgrade, we can add those back in.  I think that's a separate discussion from this PR though.